### DISCUSSION TITLE
Skip threads messages?

### DISCUSSION THREAD

**Discussion by etadex** [victornpb/undiscord#322] [1. Questions & Answers]
Hi!
I noticed when a channel has some threads archived in it, the script tries deleting messages from em, looping in the process, trying that over and over again.
Is there an option or a switch to skip threads' messages or skip messages that gave an error while deleting?

**Comment by sabotenshi** [comment]
Would love an answer to this. Alternatively, do we think we'll be able to delete convos that took place in said threads any time soon?
